packages/core/src/lib/object_factory.js

```diff
@@ -189,6 +211,22 @@ Pattern.prototype = {
   findPartial: function(partialString) {
     return this.engine.findPartial(partialString);
   },
+
+  getDirLevel: function(level, info) {
+    // eslint-disable-next-line prefer-const
+    let items = this.subdir.split(path.sep);
```

Comment:
I don't think you should be disabling eslint rules unless you have a very good reason to (and if you do that reason should probably be mentioned in a comment unless it is glaringly obvious).

Reply:
If I'm honest, I don't even know anymore why this one is there. Maybe it was a relic from the previous function I had. I will remove this one.

Reply:
Cool. You have some more on object_factory.js lines 99 and 100, but regarding conditional ternaries.